styx-scheduler-service/src/main/java/com/spotify/styx/state/handlers/TerminationHandler.java

```diff
@@ -81,7 +81,7 @@ private void checkRetry(RunState state) {
       if (state.data().lastExit().map(c -> c == MISSING_DEPS_EXIT_CODE).orElse(false)) {
         delayMillis = Duration.ofMinutes(MISSING_DEPS_RETRY_DELAY_MINUTES).toMillis();
       } else {
-        delayMillis = retryUtil.calculateDelay(state.data().tries()).toMillis();
```

Comment:
`calculateDelay` method's argument name in `RetryUtil` should be changed to `failures` just to avoid confusion